python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -52,6 +53,21 @@
     db.Column("create_timestamp", db.DateTime, server_default=get_current_timestamp()),
 )
 
+AssetEventTagsTable = db.Table(
+    "asset_event_tags",
+    SqlEventLogStorageMetadata,
+    db.Column(
+        "id",
+        db.BigInteger().with_variant(sqlite.INTEGER(), "sqlite"),
+        primary_key=True,
+        autoincrement=True,
+    ),
+    db.Column("event_id", db.Integer, db.ForeignKey("event_logs.id", ondelete="CASCADE")),
```

Comment:
hm, this may make future likely big int migration for event id a bit more challenging. Can we proactively make it bigint or does it have to match?

not having it covered by an index may make impact delete performance, especially as the table grows

Reply:
Hmm, I tried making this value a big int. It works for postgres, but MySQL requires the types to be the same. I'll keep it as a regular int for now, unless if you'd rather I make it a big int for postgres and not MySQL

What kind of index were you imagining? I noticed that we don't have one for the run tags table, perhaps we can wait until we need the index to add it?